Add rework documentation (#135)
SAP/cf-nodejs-logging-support: docs/configuration/default-req-log-level.md

---
layout: default
title: Default Request Level
parent: Advanced Configuration
nav_order: 3
permalink: /configuration/defaultrequestlevel
---

## Set default request loging level

Change the default request logging level by setting the property "reqLoggingLevel": $level in your configuration file. For example:

```ts
{
    "reqLoggingLevel": "info"
}
```

> Alternatively you can also set a default request logging level as mentioned in [Dynamic Logging Level Threshold](/cf-nodejs-logging-support/advanced-usage/dynamic-logging-level-threshold)

Reviewer: ```suggestion
Alternatively, you can set the default request logging level as mentioned in [Dynamic Logging Level Threshold](/cf-nodejs-logging-support/advanced-usage/dynamic-logging-level-threshold)
```
